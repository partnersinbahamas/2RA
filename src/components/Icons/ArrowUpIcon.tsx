import React from 'react';
import PropTypes from 'prop-types';
import Svg from './Svg';

type TProps = {
  className?: string;
  color?: string;
  style?: { [cssAttribute: string]: string };
};

/**
 * @deprecated Use ChevronUpIcon instead.
 */
const ArrowUpIcon: React.FC<TProps> = ({ className, color, style }) => (
  <Svg
    xmlns="http://www.w3.org/2000/svg"
    viewBox="5 8 22 14"
    style={style}
    fill={color}
    className={className}
  >
    <g>
      <polygon points="27,19 24,22 16,14 8,22 5,19 16,8" />
    </g>
  </Svg>
);

ArrowUpIcon.propTypes = {
  className: PropTypes.string,
  color: PropTypes.string,
  style: PropTypes.any,
};

ArrowUpIcon.defaultProps = {
  className: '',
  color: 'currentcolor',
  style: {},
};

export default ArrowUpIcon;
