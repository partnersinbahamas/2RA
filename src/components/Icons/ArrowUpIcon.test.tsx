import React from 'react';
import { render } from '@testing-library/react';
import ArrowUpIcon from './ArrowUpIcon';

test('ArrowUpIcon', () => {
  const { container } = render(<ArrowUpIcon />);
  expect(container).toMatchSnapshot();
});
