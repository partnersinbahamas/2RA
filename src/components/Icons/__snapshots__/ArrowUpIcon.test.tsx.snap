// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowUpIcon 1`] = `
<div>
  <svg
    class="sc-beySPh jroDkm"
    fill="currentcolor"
    viewBox="5 8 22 14"
    xmlns="http://www.w3.org/2000/svg"
  >
    <g>
      <polygon
        points="27,19 24,22 16,14 8,22 5,19 16,8"
      />
    </g>
  </svg>
</div>
`;
